core/modules/image/image.field.inc

```diff
@@ -458,6 +462,10 @@ function image_field_widget_form(&$form, &$form_state, $field, $instance, $langc
     backdrop_alter('image_supported_extensions', $supported_extensions);
     $extensions = isset($elements[$delta]['#upload_validators']['file_validate_extensions'][0]) ? $elements[$delta]['#upload_validators']['file_validate_extensions'][0] : implode(' ', $supported_extensions);
     $extensions = array_intersect(explode(' ', $extensions), $supported_extensions);
+    // Todo: Decide if we're keeping this permission.
+    if (!user_access('upload svg images')) {
```

Comment:
If we're defaulting to stripping out all xss then I'd say this isn't necessary.

Reply:
Removed.